src/features/encounters/npc/new/ClassCard.vue

```diff
@@ -6,6 +6,9 @@
           {{ npcc.Name }}
         </span>
       </v-col>
+      <v-col v-if="npcc.InLcp" class="ml-auto mt-n4"
+        <div class="heading h3 text--text">{{ npcc.LcpName }}</div>
```

Comment:
I played with the classes on the v-col here before in an attempt to shift it over but I can try again.